content/design/color/index.html

```diff
@@ -2,58 +2,440 @@
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-success</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#71bf43</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #fbb034"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Highlight for elements conveying an "warning" status, such as alerts, labels and toasts</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-warning</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#fbb034</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #ef4044"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Highlight for elements conveying an "danger" status, such as alerts, labels and toasts</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-danger</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#ef4044</div>
+      </div>
+    </div>
+  </div>
+  
+  <h2 class="bb-section-heading">Borders</h2>
 
-{{ include 'includes/design-table.hbs' }}
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #e2e3e4"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Borders separating sections of content within a container</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-border-color-neutral-light</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#e2e3e4</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #cdcfd2"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Borders separating containers from their surroundings, or separating sections of active elements such as a toolbar from the contents of the container</div>
```

Comment:
Delete the first comma.

I think "of" should be "within."

I'm not sure I follow the "such as" clause, but I ***think*** it should be "such as a toolbar and the contents of a container." (If the clause stays as-is, I think there should be a "from" in the text that precedes the "such as" clause.)